modules/apps/site-initializer/site-initializer-liferay-marketplace/site-initializer-liferay-marketplace-test/src/testFunctional/macros/MP2Dashboards.macro

```diff
@@ -0,0 +1,50 @@
+definition {
+
+	macro clickNewMemberButton {
+		Click(locator1 = "MP2Utils#NEW_MEMBER_BUTTON");
```

Comment:
Hey @LuanaKSilva, the path NEW_MEMBER_BUTTON can be more general with a variable, because it works in other sections, e.g. in Projects in the '+ New Project' button and in Apps in the '+ New App' button.

Reply:
Same comment as above.